kube/workflow.go

```diff
@@ -101,6 +103,50 @@ func (c *Client) create(namespace string, wf *Workflow, opts *WorkflowOptions) (
 		}
 	}
 
+	secretName := "onepanel-default-env"
+	addSecretValsToTemplate := true
+	secret, secretGetErr := c.Clientset.CoreV1().Secrets(namespace).Get(secretName, metav1.GetOptions{})
```

Comment:
I think it might be cleaner to use `err` here so it's not redefined again and then below it instead of:
```
if err, ok := secretGetErr.(*errors.StatusError); ok
```
Use the new `As` method:
```
var statusError *errors.StatusError
if errors.As(err, &statusError) {
```

Reply:
Neat to know about `As`!